pkg/image/apis/image/helper.go

```diff
@@ -41,6 +46,16 @@ const (
 	ImportRegistryNotAllowed = "registry is not allowed for import"
 )
 
+// knownSignatureStores is a list of known signature stores. The key represents
+// a registry name and it maps to a signature store for that registry. The
+// signature store has to be HTTPS and it must offer image signatures based on
+// this format: https://STORE_URL/IMAGE_NAME/signature-1
+// The response must be valid image signature blob.
+// TODO: Maybe this should be configurable via master config?
```

Comment:
Yes, probably, although it would be good to think this through fully. We're presuming for this use case the administrator controls the location of the sigstore for a set of trusted registries. I *think* that's a safe assumption, and these are likely to be very static except in test scenarios where a config override is essential.

A list of key:value pairs should work, e.g.

```
sigstores:
  - registry.access.redhat.com: "https://access.redhat.com/webassets/docker/content/sigstore"
```

NOTE: Red Hat may add a registry endpoint in the near future, and then there's registry.connect.redhat.com, which would have a separate sigstore.